packages/app/src/app/components/SignInModal.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useCallback } from 'react';
+import { ThemeProvider, Stack } from '@codesandbox/components';
+import codeSandboxBlack from '@codesandbox/components/lib/themes/codesandbox-black';
+import OutsideClickHandler from 'react-outside-click-handler';
+import { SignInModalElement } from 'app/pages/SignIn/Modal';
+import { useOvermind } from 'app/overmind';
+import { motion, AnimatePresence } from 'framer-motion';
+
+export const SignInModal = () => {
+  const {
+    actions: { toggleSignInModal },
+    state: { redirectOnLogin },
+  } = useOvermind();
+
+  const closeModal = useCallback(
+    event => {
+      if (event.keyCode === 27 && open) toggleSignInModal();
```

Comment:
Put curly braces around body + use `ESC` constant

```suggestion
      if (event.keyCode === ESC && open) {
        toggleSignInModal();
      }
```